examples/demo-rollup/benches/prover/README.md

```diff
@@ -171,4 +171,56 @@ Total cycles consumed for test: 20834815
 Total cycles consumed for test: 26152702
 ```
 * There's an overall efficiency of 6 million cycles in total for 3 blocks. 
-* Keep in mind that the above table shows average number of cycles per call, so they give an efficiency per call, but the "Total cycles consumed for test" metric at the bottom shows total for 3 blocks
\ No newline at end of file
+* Keep in mind that the above table shows average number of cycles per call, so they give an efficiency per call, but the "Total cycles consumed for test" metric at the bottom shows total for 3 blocks
+
+* With ed25519 acceleration
+```
++----------------------+----------------+-----------+
```

Comment:
I've asked ChatGPT to merge these 2 tables.

In my opinion it improves readability:


| Function | Avg Cycles w/o Accel | Avg Cycles With Accel | % Change | Num Calls |
| --- | --- | --- | --- | --- |
| Cycles per block | 4,764,675 | 1,684,534 | \-64.65% | 3 |
| apply\_blob | 3,979,880 | 899,771 | \-77.39% | 3 |
| verify | 3,579,797 | 714,955 | \-80.03% | 3 |
| end\_slot | 413,717 | 415,147 | +0.35% | 3 |
| compute\_state\_update | 393,992 | 397,247 | +0.83% | 3 |
| begin\_slot | 83,817 | 82,357 | \-1.74% | 3 |
| commit | 7 | 7 | 0.00% | 3 |
| **Total** | **13,215,885** | **4,194,018** | **\-68.27%** |  |

Reply:
thats a very good idea ! making that change